### PR TITLE
Obj store through sidecar

### PR DESCRIPTION
**Pull request by jdoliner**

### CODE REVIEW

File: src/client/pfs/pfs.proto [hunk 1]
```diff
@@ -664,6 +664,15 @@ message Objects {
   repeated Object objects = 1;
 }
 
+message PutObjRequest {
```
[the remark this answers is not in the document]
Comment (jdoliner) [reply src/client/pfs/pfs.proto]: Fair point I can do that.

**Comment by jdoliner** [comment]
What did you have in mind for tests of this behavior specifically. Other than trying to come up with containers that reproduce the original issue, which I'm kind of unsure how to do, it seems like our existing pipeline tests are basically the same as whatever tests I would write.

**Comment by Tryneus** [comment]
After a quick offline discussion, I withdraw my objection for testing until such time as we have a better mock object storage client framework for testing.

**Comment by jdoliner** [comment]
Renames done.

**Comment by Tryneus** [comment]
LGTM

**Pull request merged**